src/Internal/DependencyManagement/CoreServiceProvider.php

```diff
@@ -279,6 +280,7 @@ function ( ...$arguments ) {
 		$this->share_with_tags( PHPViewFactory::class );
 
 		// Share other classes.
+		$this->conditionally_share_with_tags( Activated::class );
 		$this->conditionally_share_with_tags( Loaded::class );
```

Comment:
Unrelated to this PR but this `Loaded` event has the tag `Demo Class` on its doc block 😅and I can't find any place that calls it. Can we remove it maybe?

Reply:
I believe this was the early example/template event. But since we already have several _real_ events now, this can be ✂️  4d7e19366c6ba9325463a58a73ccdfe5a07e57ff